src/test/java/CalculatorTest.java

```diff
@@ -0,0 +1,39 @@
+import org.junit.Assert;
+import org.junit.Test;
+import taskAT7.Calculator;
+
+public class CalculatorTest {
+
+    @Test
+    public void add() {
+        Calculator calculator = new Calculator();
+        int result = calculator.add(2, 4);
```

Comment:
maybe 'actualResult' then? why this values were selected as an input? Based on which technique were they set?